setup.py

```diff
@@ -218,7 +218,8 @@ def run(self):
     ':python_version == "2.7" or python_version == "3.3"': ['pathlib2'],
     ':sys_platform != "win32"': ['pexpect'],
     ':sys_platform == "darwin"': ['appnope'],
-    ':sys_platform == "win32"': ['colorama', 'win_unicode_console>=0.5'],
+    ':sys_platform == "win32"': ['colorama'],
+    ':sys_platform == "win32" and python_version < "3.6"': ['win_unicode_console>=0.5'],
```

Comment:
I'm guessing that since setuptools 17 is more than a year ago, I suspect that most people with setuptools < 17 have setuptools 0.6 + pip 1.5.6, and need to upgrade anyway. I doubt many folks are stuck on setuptools 16.